.github/workflows/docs-build.yml

```diff
@@ -0,0 +1,126 @@
+name: Docs Build Workflow
+
+on:
+  push:
```

Comment:
Let's do this for pull requests only.  The automation account will only push if documentation builds are successful, so this is not needed.